From: Dima Kogan <dima@secretsauce.net>
Date: Fri, 20 Nov 2015 21:05:22 -0800
Forwarded: https://github.com/sukria/Backup-Manager/issues/65
Subject: Documentation of the documentation cleanup

- updated doc/README to indicate how to make the docs. This told the user to hit
  a no-longer-existent website
---
 doc/README | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/doc/README b/doc/README
index e8d3f44..c7a71c1 100644
--- a/doc/README
+++ b/doc/README
@@ -1,8 +1,3 @@
-The text version of the user guide is provided here:
-user-guide.txt
-
-If you want to use the HTML or PDF format, grab it from the Backup
-Manager website:
-http://www.backup-manager.org/documentation/
-
-This has been written in SGML with the debiandoc tools.
+The documentation is shipped as an SGML document. To build the PDF, TXT and HTML
+versions of the documentation, run a "make" here. This uses tools debiandoc2xxx
+from the "debiandoc-sgml" debian package
